db/app.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express')
+const app = express()
+
+
+const {getTopics}  = require('./controllers');
+app.use(express.json())
```

Comment:
Arguably, this doesn't need to be included until you're making a post or patch request - if you were to take line 6 out, your GET request would still work, as app.use(express.json()) enables us to receive request bodies, which we don't need for a GET request. This is fine to stay in, though it's important to understand it's purpose.